### DISCUSSION TITLE
[Bug]: InvalidPassword / InvalidPassword on logging in to Steam.

### DISCUSSION THREAD

**Discussion by whyynoot** [SteamRE/SteamKit#1346] [General]
### What did you expect to happen?

Expected log in success event, I may do something wrong 

### Instead of that, what actually happened?

Getting InvalidPassword in SteamUser.LoggedOnCallback callback, but I am 100% that the password is valid. Tried to use a lot of accounts. same issue 

### Which operating system are you running on?

Windows

### Which .NET Runtime are you running on?

.NET Framework

### Version

4.7.2

### Example Code

```csharp
var steamUser = steamClient.GetHandler<SteamUser>();
 var manager = new CallbackManager(steamClient);
 manager.Subscribe<SteamUser.LoggedOnCallback>(OnLoggedOn);
authSession = await steamClient.Authentication.BeginAuthSessionViaCredentialsAsync(new AuthSessionDetails
         {
             Username = username,
             Password = password,
             IsPersistentSession = false,
             Authenticator = new UserFormAuthenticator(targetAccount),
         });

var pollResponse = await authSession.PollingWaitForResultAsync();

 steamUser.LogOn(new SteamUser.LogOnDetails
 {
     Username = pollResponse.AccountName,
     AccessToken = pollResponse.RefreshToken,
 });
 while (running)
 {
     manager.RunWaitCallbacks(TimeSpan.FromSeconds(1));
 }
```

### Additional Information

I am sorry if this is a duplicate or something. I may do something worng

**Comment by w4559w** [comment]
I want to know too

**Comment by userzd0907** [comment]
I want to know too
